src/storage/src/hummock/table_accessor.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use risingwave_hummock_sdk::HummockSSTableId;
+
+use crate::hummock::sstable_store::{SstableStoreRef, TableHolder};
+use crate::hummock::HummockResult;
+use crate::monitor::StoreLocalStatistic;
+
+#[async_trait::async_trait]
+pub trait TableAccessor: Clone + Sync + Send {
```

Comment:
Need more docs here. Should also mention SstableStoreRef also impl this trait.

Reply:
I have removed `TableAccessor`.